common_benchmark_suite/openxla/benchmark/comparative_suite/jax/scripts/generate_model_artifacts.py

```diff
@@ -119,14 +138,22 @@ def main(output_dir: pathlib.Path, filter: str, iree_opt_path: pathlib.Path):
                      f' Available models:\n{all_models_list}')
 
   output_dir.mkdir(parents=True, exist_ok=True)
+
   for model in models:
     # We need to generate artifacts in a separate proces each time in order for
     # XLA to update the HLO dump directory.
     p = multiprocessing.Process(target=_generate_artifacts,
-                                args=(model, output_dir, iree_opt_path))
+                                args=(model, output_dir, iree_opt_path,
+                                      auto_upload))
     p.start()
     p.join()
 
+  if auto_upload:
+    subprocess.run(
```

Comment:
Maybe we can create a helper function for this to dedup the code?

Reply:
Created a utils.gcs_upload() function.